structopt-derive/src/attrs.rs

```diff
@@ -76,6 +76,13 @@ impl Attrs {
                 self.methods = methods.into_iter().filter(|m| m.name != name).collect();
             }
             ("name", new_name) => self.name = new_name.into(),
+            ("validator", _) => {
```

Comment:
I don't want special casing for normal method.

Reply:
This is where we map from key/value attribute pairs (where the value is a string) to methods. If we don't special case here, we'd have to special case somewhere else. The `arg` to  the `validator` method is an identifier (or a closure, but that should be handled somewhere else). 

In other words, I'm not sure how else you would expect to handle this.